### ISSUE TITLE
Add PRONE Mode

### ISSUE THREAD

**Issue by nihal111** [MovingBlocks/Terasology#2556]
#2544 adds the crouch mode.
The prone mode needs to be added, similarly.
Key-mapping and other relevant discussion can be found [here](http://forum.terasology.org/threads/crouch-and-prone-controls.1622/).

**Comment by nihal111** [comment]
I would like to work on this. However I need to know how I could create a capsule collider that has its axis along the ground. As shown in the image: 
![stance](https://cloud.githubusercontent.com/assets/13916513/19548718/c9a9d70a-96bd-11e6-9781-9ed7e607e988.png)

**Comment by emanuele3d** [comment]
From your graphics it appear that in crouching position not only the height but also the width and (presumably) depth of the capsule are shrunk proportionally. Is that the case currently?

**Comment by nihal111** [comment]
@emanuele3d No. Only height changes as of now. Pardon my lack of image-editing skills :stuck_out_tongue:

**Comment by emanuele3d** [comment]
I thought so. I just wanted to make sure. :smile:
